### ISSUE TITLE
xyBriToRgb returns number greater than 255

### ISSUE THREAD

**Issue by liamse** [Shnoo/js-CIE-1931-rgb-color-converter#7]
Following example will return number greater than 255 for Green.

```js
let rgb = ColorConverter.xyBriToRgb(0.735 ,0.2645 , 255);
// rgb = {r: 255, g: 1020, b: 43}
```

The problem is you check negative number as well
```js
let red = parseInt(r * 255) > 255 ? 255: parseInt(r * 255);
let green = parseInt(g * 255) > 255 ? 255: parseInt(g * 255);
let blue = parseInt(b * 255) > 255 ? 255: parseInt(b * 255);
```

**Comment by bokub** [comment]
Should be fixed by #5 

I'm not sure my PR will ever be merged, but meanwhile, you can use the code on [my branch](https://github.com/bokub/js-CIE-1931-rgb-color-converter/tree/patch-1), or use [this CDN URL](https://cdn.jsdelivr.net/gh/bokub/js-CIE-1931-rgb-color-converter@patch-1/ColorConverter.min.js).

**Comment by Shnoo** [comment]
Merged #5
